docs/builders/ephemeris/horizons-gui.md

```diff
@@ -0,0 +1,83 @@
+
+## File Path
+Here you specify where on your computer you want the data to be stored once it has been downloaded. You can click the *Browse* button to browse with the file explorer to specify where the data should be stored. This is required to be specified for the data to be downloaded. If there was any issue with the download, then there will instead be an error file stored at this location. The error file will be named <code>XXX_error.txt</code> where <code>XXX</code> is the name of the file you specified. If you adjust your input to fix this issue and the download succeeds this time, this error file will be deleted automatically.
+
+## Target Body
+Here you can type in the name of the thing you want data for. For example if you want data for the Voyager 1 spacecraft you can write <code>Voyager 1</code> here. Horizons have data for many different things from spacecraft, planets, moons, comets and asteroids to lagrange points and more. In this field and the **Observer Location** field the characters <code>¤ < > § £ ´ ¨ €</code> and other special characters from other languages are not allowed. Additionally the characters <code>' =</code> have some limitations in how they can be used.
+
+## Observer Location
+Here you can type in the name of the reference point you want for the data. For example if you want to show the movement of the spacecraft Voyager 1 over time, the reference point should probably be the Sun or the Solar System Barycenter. But if you instead want to show a satellite that orbits around Mars, the reference point should probably be Mars.
+
+## Start Time
+Here you specify the start of the time range you want data for.
+
+## Stop Time
+Here you specify the end of the time range you want data for.
+
+## Step Size
+Here you can specify at what resolution you want the data to be in. You start with a number and then you can choose a time unit in the drop down menu to the right.
+
+## Horizons log messages
+This is a log that only displays messages related to your input in this tool and the download of the data. When using this tool there are a few different issues that you can encounter and they will be explained in more detail below. All of the issues you will encounter will be detailed in this log and the messages will guide you towards a fix. If OpenSpace fails with getting this helpful information there will be an error file stored at the specified location in the **File Path** field. This file might contain some useful information about what went wrong and how it can be fixed.
+
+
+# Issues and Solutions
+There are a few different things that can go wrong while using this tool. Most of the issues are related to the input and can be fixed with some adjustments. The particular parameters that need to be adjusted will be highlighted in red and there will be some messages in the Horizons log that can give you tips on how to fix it.
```

Comment:
Add a sentence, för "röd tråd"

"Some common errors and solutions for those errors are explained below"